biz.aQute.bndlib/src/aQute/bnd/build/api/package-info.java

```diff
@@ -0,0 +1,8 @@
+/**
+ */
+
+@Version("4.0.0")
+@org.osgi.annotation.bundle.Export
```

Comment:
We should not be using `@Export` in our package-infos. It messes something out build up. We should modify the `-exportcontents` in the bnd file.

Reply:
ok ... a pity